kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/embedder/MavenProjectLoader.java

```diff
@@ -68,9 +78,7 @@ public static MavenProject parseMavenPom(File pomFile, boolean offline) {
         MavenEmbedder mavenEmbedder = null;
         try {
             mavenEmbedder = new MavenEmbedder(mavenRequest);
-            return hasPom ?
-                    mavenEmbedder.readProject(pomFile) :
-                    mavenEmbedder.readProject(new ByteArrayInputStream(DUMMY_POM.getBytes(StandardCharsets.UTF_8)));
+            return hasPom ? mavenEmbedder.readProject(pomFile) : mavenEmbedder.readProject(new ByteArrayInputStream(DUMMY_POM.getBytes(StandardCharsets.UTF_8)));
```

Comment:
is this only format?

Reply:
only format - in this change I had to change the pom in my tests to point to a one with a transitive dep and then check if the transitive dep was resolved - and also a test without the transitive dep - so switching the pom helped to make both tests